spyro/receivers/changing_coordinates.py

```diff
@@ -763,4 +763,4 @@ def change_to_reference_hexa(p, cell_vertices):
     # pny = px * a21 + py * a22 + pz * a23 + a24
     # pnz = px * a31 + py * a32 + pz * a33 + a34
 
-    return (pnx, pny, pnz)
+    # return (pnx, pny, pnz)
```

Comment:
Here I think that removing the big block of comments is more appropriate to make the code easier to navigate.

Reply:
I have temporarily removed this section in the latest commit. However, I plan on adding an improved version if we ever work with hexahedrals not based on extruded meshes. Thanks for the suggestion!